src/rickbox/blueprints/db.py

```diff
@@ -0,0 +1,85 @@
+import re
+import warnings
+import click
+import pymysql
+from flask import Blueprint
+from database import DatabaseHelper
+from database.sql.procs import get_all_procs
+from database.sql.schema import get_all_schemas
+
+db_bp = Blueprint('db', __name__)
+
+
+@db_bp.cli.command('init')
+def init():
+    """ Initializes the database tables, stored procedures, associations, etc. """
+    print('Loading schemas...')
+    for query_set in [get_all_schemas(), get_all_procs()]:
+        for key in query_set.keys():
+            print(f'Creating {key}...')
+            print(query_set[key])
+            queries = str(query_set[key])
+            queries = re.sub(r'(\n| )+', ' ', queries)
+            queries = queries.split(';')
+            queries = [query for query in list(map(str.strip, queries)) if len(query) > 0]
+            for query in queries:
+                print('lenquery:\t', len(query))
+                def operation():
+                    with DatabaseHelper() as db:
+                        db.execute(query)
+
+                handle_warnings(operation, key)
+    print('Done!')
+
+
+@db_bp.cli.command('drop')
+@click.option('--force', is_flag=True, help='This is a safety mechanism')
+@click.option('--db', required=True, help='The name of the database to drop')
+def drop(force, db):
+    """ Drops the database. Be careful. """
+    if not force:
+        print('The --force flag was not passed. No action taken')
+        return
+    make_or_drop(db, 'drop')
```

Comment:
I put some safety rails on this, but could be argued it shouldn't exist at all.
Must be called via `flask db drop --db=DBNAMEHERE --force` or it won't work. Should not be possible to call it by accident.